dali/pipeline/executor/executor.cc

```diff
@@ -83,8 +83,8 @@ void Executor::RunCPU() {
   // Run the support ops
   try {
     WorkspaceBlob &wsb = wss_[queue_idx];
-    for (int i = 0; i < graph_->NumSupportOp(); ++i) {
-      OpNode &op_node = graph_->support_node(i);
+    for (int i = 0; i < graph_->NumOp(DALIOpType::DALI_SUPPORT); ++i) {
```

Comment:
If you _have to_ use enum class, then take full advantage of it and remove `DALI_` prefix from each and every field.

Reply:
Done in #597